Cargo.toml

```diff
@@ -34,7 +34,11 @@ byteorder = "1.4.3"
 thiserror = "1.0"
 halo2curves = { version = "0.4.0", features = ["derive_serde"] }
 group = "0.13.0"
+tap = "1.0.1"
```

Comment:
Should these be dev dependencies? I see that `tap` is used in test.rs.

Reply:
Will tighten that up.